test/e2e/invconfig/configmap.go

```diff
@@ -37,18 +36,14 @@ func NewConfigMapTypeInvConfig(cfg *rest.Config) InventoryConfig {
 func newDefaultInvApplierFactory(cfg *rest.Config) applierFactoryFunc {
 	cfgPtrCopy := cfg
 	return func() *apply.Applier {
-		return newApplier(inventory.ClusterClientFactory{
-			StatusPolicy: inventory.StatusPolicyAll,
-		}, cfgPtrCopy)
+		return newApplier(inventory.ClusterClientFactory{}, inventory.StatusPolicyNone, cfgPtrCopy)
```

Comment:
hmm. Does the ConfigMap inv client work with StatusPolicyAll? Obviously it doesn't need it, since there's no status... But would doing so make the client error when trying to update the status?

Reply:
That is correct, the client would error if you tried to update the status while using ConfigMap inventory

Reply:
> if you tried to update the status

Does the ConfigMap Inventory error if you pass the StatusPolicyAll to the Applier?

Reply:
Yes, it would error when trying to update the status

Reply:
I think this is fine, as long as we document somewhere obvious that when using the ConfigMapInventory, you need to also use StatusPolicyNone.